hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/XceiverServerGrpc.java

```diff
@@ -69,6 +69,7 @@
   private UUID id;
   private Server server;
   private final ContainerDispatcher storageContainer;
+  private volatile boolean isStarted;
```

Comment:
Sorry missed this do we need volatile here?

Reply:
I thought this will be multi-threaded. Given this is single threaded, volatile is not needed. I will remove it.